bower.json

```diff
@@ -31,7 +31,8 @@
     "angular-trix": "^1.0.2",
     "angular-moment": "^1.2.0",
     "moment": "^2.21.0",
-    "ngclipboard": "^2.0.0"
+    "ngclipboard": "^2.0.0",
```

Comment:
Since you are adding new dependencies, it would be great if you can drop your node_modules folder and try building it again for the sanity check.